frontend/src/components/Home.vue

```diff
@@ -224,7 +224,7 @@ export default {
           img: require('../assets/analyze.jpg'),
           route: { name: 'viewer',
             params: { model: 'Human-GEM', type: 'compartment', map_id: 'endoplasmic_reticulum' },
-            query: { dim: '2d', panel: '1', g1: 'skin 1', coords: '-7222.7,-4501.6,0.97,0,0,0' } },
+            query: { dim: '2d', panel: '1', coords: '-7222.7,-4501.6,0.97,0,0,0' } },
```

Comment:
Do you know what is this `g1: skin 1` for @mihai-sysbio @inghylt ?

Reply:
It was the url param `skin 1` for the first tissue `1` for gene expression `g`.

Reply:
Thanks for the explanation, but it seems this is not in use now?

Reply:
Exactly, that is why it was removed! All relevant url params for data overlay are now grouped through `queryParams` in  `frontend/src/store/modules/dataOverlay.js`